src/components/Nav/Nav.css

```diff
@@ -0,0 +1,27 @@
+@import url("https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800");
+.navbar {
+  display: flex;
+  justify-content: space-between;
+}
+
```

Comment:
Can you rename this component something like DashboardLinks? Just to save confusion with the actual 'Nav' components, i.e. the breadcrumb (which we haven't created yet)

Reply:
the navbar does not contain any of the dashboard links